ui/public/lib/api-client.js

```diff
@@ -1,5 +1,4 @@
 // @ts-check
-/* globals window, WebSocket */
```

Comment:
Looks to me like `window` at least is still used as a free variable here. Our tools are supposed to at least warn on such variables without this "globals" declaration. This is directly related to the outstanding bug on too much ambient authority.

Do our tools not warn if you get rid of this "globals" directive? That itself is bug. Please file an issue.

Reply:
Not sure that tool is attached to this repo

Reply:
How can we ensure greater tool uniformity among our repos?

Reply:
I don't know any short path to get there.

_And I think we've drifted pretty far off topic from this PR..._

Reply:
> And I think we've drifted pretty far off topic from this PR...

Agreed!